test/default_route_test.rb

```diff
@@ -29,10 +29,10 @@
 
     allow(Yast::SCR).to receive(:Read) do |path|
       if path.to_s =~ /^\.network\.value\."(eth\d+)".(.*)/
-        @ifcfg_files.get(Regexp.last_match(1), Regexp.last_match(2))
-      else
-        raise "Unexpected Read #{path}"
+        next @ifcfg_files.get(Regexp.last_match(1), Regexp.last_match(2))
```

Comment:
in block next is in its context like local return. So this next do quick return of this value.

It is similar like if you have method
```
def m
  if a
    return val
  elsif b
     return valb
  end

  raise "unknown value"
end
```